oeis.py

```diff
@@ -170,6 +170,23 @@ def A000040(start: int = 0, limit: int = 20) -> Collection[int]:
 
 
 @oeis
+def A023811(start, limit):
```

Comment:
According to the travis report, you should add type annotation here, as all other functions did.